pkg/keys/v2/evm_tx.go

```diff
@@ -0,0 +1,154 @@
+// Package keys/v2 provides key management functionality for EVM transactions and OCR2 keyrings.
+package keys
+
+import (
+	"context"
+	"errors"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind/v2"
+	"github.com/ethereum/go-ethereum/common"
+	gethtypes "github.com/ethereum/go-ethereum/core/types"
+	gethcrypto "github.com/ethereum/go-ethereum/crypto"
+	"github.com/smartcontractkit/chainlink-common/keystore"
+)
+
+const (
+	// EVMPrefix is the prefix for EVM-related keys.
+	EVMPrefix = "evm"
+	// TxKeystorePrefix is the prefix for transaction keys.
+	TxKeystorePrefix = "tx"
```

Comment:
Will these be easier to find if they share a prefix instead of a suffix?
```suggestion
	PrefixEVM = "evm"
	// TxKeystorePrefix is the prefix for transaction keys.
	PrefixTxKeystore = "tx"
```

Reply:
sure